moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/collision_detector_allocator_bt.h

```diff
@@ -0,0 +1,55 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2019, Open Source Robotics Foundation, Inc.
```

Comment:
This can just be your name. Not clear to me why you are listing OSRF since really GSoC is funding this project and Bryce/Felix are mentoring it.

Reply:
I thought it had to be a company